apps/openmw/mwmechanics/actors.cpp

```diff
@@ -374,9 +374,13 @@ namespace MWMechanics
             if (LOS)
             {
                 MWBase::Environment::get().getMechanicsManager()->startCombat(actor1, actor2);
-                if (!againstPlayer) // start combat between each other
+                if (!againstPlayer) // start combat between each other if engaging actor is not one of the player followers
```

Comment:
> start combat between each other if engaging actor is not one of the player followers

That's not how vanilla works, AFAIK. Can you think of a different solution to the problem?